### DISCUSSION TITLE
Rejected due to being off campus

### DISCUSSION THREAD

**Discussion by Benjamin-Lee** [community/community#129708] [GitHub Education]
### Hi everyone,

I'm a weird kind of PhD student. I split my time between the US and the UK as part of the NIH OxCam scholars program. It's a [requirement of the program](https://oxcam.gpp.nih.gov/about/phd-program). You can see me listed as a current student on an [**official US government website**](https://oxcam.gpp.nih.gov/current-students/student-profiles?combine=Benjamin+Lee&category_id=All&ic_id=77&scholar_type_id=All&first_name=&last_name=). I submitted this information along with a dated copy of my Oxford current student status report and it was still rejected. I was approved in 2021 and 2022 despite being in the exact same program. I'm not sure what changed but I was hoping someone could let me know what's going on.

Thanks,
Benjamin

**Comment by Frosty-J** [comment]
No promises this will work if you're in a different country, but the location used for the application seems entirely client-side. So you can spoof it, e.g. https://beebom.com/fake-geo-location-google-chrome-mozilla-firefox-microsoft-edge/
